### ISSUE TITLE
Slider label is read out frequently

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/molecules-and-light#237]
For some reason VoiceOver is repeating the slider label in this sim frequently. See https://github.com/phetsims/molecules-and-light/issues/234#issuecomment-564062693.

I will investigate.

**Comment by jessegreenberg** [comment]
Before it reads the name again VO makes the sound that indicates focus has moved. So I am wondering if the PDOM is getting redrawn and causing this.

**Comment by jessegreenberg** [comment]
The slider is not getting redrawn but the screen summary gets redrawn frequently.

**Comment by jessegreenberg** [comment]
I verified that every time a new element is created and added to the DOM VoiceOver re-reads the slider name.

**Comment by jessegreenberg** [comment]
HTML Validator at https://validator.w3.org/nu/#textarea is reporting no issues. So this seems to be a problem with updating the PDOM so frequently.

**Comment by jessegreenberg** [comment]
The reason we are is recreating PDOM elements and re-sorting the DOM so frequenlty in this sim is because we are using `accessibleName` setter so frequently. And that is why we aren't seeing this issue in any other issues.

**Comment by jessegreenberg** [comment]
@terracoda just so you know, I discovered that this is a general scenery problem rather than an issue with the slider (https://github.com/phetsims/scenery/issues/1026)

For now, I think I can fix this in molecules-and-light by removing usages of `setAccessibleName`.

**Comment by terracoda** [comment]
Ok, I guess it is new problem because this is the first time I have noticed it.

**Comment by jessegreenberg** [comment]
Correct, I verified that molecules-and-light is the first sim to have the conditions necessary to cause this bug.

**Comment by jessegreenberg** [comment]
After the above commits I am no longer seeing this issue. @terracoda can you please confirm?

**Comment by terracoda** [comment]
The label is only read once now. Sounds good, thanks @jessegreenberg !

**Comment by samreid** [comment]
There are still TODOs marked for this issue, discovered during https://github.com/phetsims/chipper/issues/946

**Comment by terracoda** [comment]
@jessegreenberg, I don't this issue was re-opened for me. Assigning to you. Please re-assign me when needed?

**Comment by samreid** [comment]
The TODO is:

```js
    Property.multilink( summaryProperties, ( photonWavelength, emitterOn, photonTarget, running, slowMotion, returnMoleculeButtonVisible ) => {

      // TODO: Maybe use accessibleName instead if https://github.com/phetsims/molecules-and-light/issues/237 is fixed
      dynamicDescription.innerContent = this.getSummaryString();
    } );
```

**Comment by samreid** [comment]
Since https://github.com/phetsims/molecules-and-light/issues/237 was closed, perhaps this should be changed to use `accessibleName`?

**Comment by jessegreenberg** [comment]
Thanks @samreid, the real issue that needs to be fixed before the TODO can be removed is https://github.com/phetsims/scenery/issues/1026, so I updated the code comment to reference that open issue instead.
